### PR TITLE
Mark multiple messages as (not) important

### PR DESCRIPTION
**Pull request by GretaD**

### CODE REVIEW

File: src/components/EnvelopeList.vue [hunk 1]
```diff
@@ -26,6 +26,36 @@
...
+							)
+						}}
+					</ActionButton>
+					<ActionButton icon="icon-important"
+						:close-after-click="true"
+						@click.prevent="markSelectionUnimportant">
+						{{
+							n(
+								'mail',
+								'Mark {number} as  unimportant',
```
Comment (ChristophWurst) [inline src/components/EnvelopeList.vue:50]:
```suggestion
								'Mark {number} as unimportant',
```

File: src/components/EnvelopeList.vue [hunk 2]
```diff
@@ -214,6 +244,13 @@ export default {
 			// returns false if at least one selected message has not been read yet
 			return this.selectedEnvelopes.every((env) => env.flags.seen === true)
 		},
+		areAllSelectedImportant() {
```
Comment (ChristophWurst) [inline src/components/EnvelopeList.vue:247]: unused

**Review by ChristophWurst — APPROVED** [review]

**Review by st3iny — APPROVED** [review]
I tested and it works. 

One minor thing: Both buttons are always shown, regardless if I selected important and/or unimportant messages. E.g. I select only unimportant envelopes and I'm still presented with the option to mark them unimportant.

**Comment by st3iny** [comment]
I fixed the button issue I was referring to in my review. Now the button get shown if at least one envelope matches their criteria.

File: src/components/EnvelopeList.vue (hunk 2 above)
Comment (st3iny) [reply src/components/EnvelopeList.vue]: not anymore

**Comment by GretaD** [comment]
> I fixed the button issue I was referring to in my review. Now, the buttons are shown if at least one envelope matches their criteria.

you are great <3

**Pull request merged**